diff -uprN web-6.0.1/scripts/js/footer.js web-6.0.1.cust/scripts/js/footer.js
--- web-6.0.1/scripts/js/footer.js	2025-02-21 00:07:25.000000000 +0100
+++ web-6.0.1.cust/scripts/js/footer.js	2025-02-21 11:19:08.925063131 +0100
@@ -498,7 +498,7 @@ function updateVersionInfo() {
         branch: version.core.local.branch,
         hash: version.core.local.hash,
         hash_remote: version.core.remote.hash,
-        url: "https://github.com/pi-hole/pi-hole/releases",
+        url: "https://aur.archlinux.org/packages/pi-hole-core",
       },
       {
         name: "FTL",
@@ -507,7 +507,7 @@ function updateVersionInfo() {
         branch: version.ftl.local.branch,
         hash: version.ftl.local.hash,
         hash_remote: version.ftl.remote.hash,
-        url: "https://github.com/pi-hole/FTL/releases",
+        url: "https://aur.archlinux.org/packages/pi-hole-ftl",
       },
       {
         name: "Web interface",
@@ -516,7 +516,7 @@ function updateVersionInfo() {
         branch: version.web.local.branch,
         hash: version.web.local.hash,
         hash_remote: version.web.remote.hash,
-        url: "https://github.com/pi-hole/web/releases",
+        url: "https://aur.archlinux.org/packages/pi-hole-web",
       },
     ];
 
